drivers/sensor/tmp112/tmp112.c

```diff
@@ -98,8 +114,7 @@ static int tmp112_attr_set(const struct device *dev,
 			return -ENOTSUP;
 		}
 
-		if (tmp112_reg_update(drv_data, TMP112_REG_CONFIG,
-				      TMP112_EM_BIT, value) < 0) {
+		if (tmp112_update_config(dev, TMP112_CONFIG_EM, value) < 0) {
```

Comment:
Is this part (and related call) belonging to multi-instance commit? Or is it part of the configuration change?

Reply:
It's actually a bit of both, I will try to reflect that in the commit separation